graph-api.yaml

```diff
@@ -0,0 +1,463 @@
+                operationName:
+                  type: string
+                variables:
+                  type: object
+                extensions:
+                  type: object
+              required:
+                - query
+
+  responses:
+      GraphQLSuccess:
+        description: Successful response containing query results.
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                data:
+                  type: object
+      GraphQLError:
+        description: Error response for invalid GraphQL requests, adhering to the GraphQL error specification.
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                errors:
+                  type: array
+                  description: A list of errors encountered during query execution.
+                  items:
+                    type: object
+                    properties:
+                      message:
+                        type: string
+                        description: A human-readable description of the error.
+                      locations:
+                        type: array
+                        description: A list of locations in the request document associated with the error.
+                        items:
+                          type: object
+                          properties:
+                            line:
+                              type: integer
+                              description: Line number in the request document where the error occurred.
+                            column:
+                              type: integer
+                              description: Column number in the request document where the error occurred.
+                      path:
+                        type: array
+                        description: A path to the field that caused the error.
+                        items:
+                          type: string
+                      extensions:
+                        type: object
+                        description: Additional metadata about the error, as provided by the server.
+              required:
+                - errors
+
+  schemas:
+    ValidationReport:
+      description: Validation report returned in RDF form, representing the results of SHACL validation.
+      content:
+        text/turtle:
+          schema:
+            type: string
+            description: The validation report serialized in Turtle format.
+      examples:
+        text/turtle:
+          summary: Example Turtle validation report.
+          value: |
+            PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> 
+            PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#> 
+            PREFIX sh: <http://www.w3.org/ns/shacl#> 
+            [ 
+              rdf:type sh:ValidationReport; sh:conforms true 
+            ] .
```

Comment:
This doesn't seem to be valid when tested at https://editor-next.swagger.io/
Suggest the following instead:
```
ValidationReport:
      description: Validation report returned in RDF form, representing the results of SHACL validation.
      example: |
        PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
        PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>\
        PREFIX sh: <http://www.w3.org/ns/shacl#>
        
        [ rdf:type sh:ValidationReport;
          sh:conforms true ] .
```

Reply:
Interesting. I was using https://editor.swagger.io/ which works slightly differently. Gives a similar warning admittedly.